internal/resolution/pm/nuget/job.go

```diff
@@ -62,3 +66,15 @@ func (j *Job) runInstallCmd() ([]byte, error) {
 
 	return installCmdOutput, nil
 }
+
+func (j *Job) cleanupTempoCsproj() {
```

Comment:
never seen temporary being abbreviated to anything else :D

```suggestion
func (j *Job) cleanupTempCsproj() {
```